src/lib/snack-bar/snack-bar.md

```diff
@@ -72,3 +72,7 @@ export class MessageArchivedComponent {
   constructor(@Inject(MD_SNACK_BAR_DATA) public data: any) { }
 }
 ```
+### Accessibility
+A snack-bar automatically goes away after a short time, so you can't count on the user seeing the
```

Comment:
I would phrase this as
```markdown
Snack-bar messages are announced via an `aria-live` region. Focus is not moved to
the snack-bar element, as this would be disruptive to a user in the middle of a
workflow. For any action offered in the snack-bar, the application should offer the
user an alternative way to perform the action (typically via keyboard shortcut).
```